shared/chat/inbox/container/index.js

```diff
@@ -125,6 +128,16 @@ export default compose(
           nextProps.onUntrustedInboxVisible(toUnbox)
         }
       }
+
+      // keep first item selected if filter changes
+      if (!isMobile) {
```

Comment:
as you type the filter always keep the first thing selected, unless you arrow around